### PR TITLE
Assignement 4 Natalia Figal

### PR DESCRIPTION
**Pull request by nataliafigal**

### CODE REVIEW

**Comment by SerPablo** [comment]
Comments, 

please finish the exercises such as 7.5. Hint for 7.5:
?Subject xxxxxxxxxxxxxxxxxxxxx. xxxxxxxx
filter(?b != ?c)

Please check files and folders to be merged. In this pull request there are two folders. Remove one. Files have to be reflected in Assignment4/{name_or_id}/{files}
Task9 should not be in this pull request

**Comment by SerPablo** [comment]
Please complete task7 in both rdflib and SPARQL.

File: Assignment4/natalia figal- 22C051/copia_de_task06.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the email address, fullName, given and family names**"""
+
+from rdflib import XSD
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, VCARD.EMAIL, Literal("jane_smith@example.org")))
+g.add((ns.JaneSmith, VCARD.GIVEN, Literal("Jane")))
```
Comment (dgarijo) [inline Assignment4/natalia figal- 22C051/copia_de_task06.py:64]: GIVEN is not in vcard (see capitalization). 
Also, family name (line below) is not there. Search for the right properties

File: Assignment4/natalia figal- 22C051/copia_de_task07.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  initNs = { "class_of_interest": class_of_interest}
+)
+
+for r in g.query(q1):
+  print(r.subclass)
+
+"""**TASK 7.2: List all individuals of "Person" with RDFLib and SPARQL (remember the subClasses)**
+
+"""
+
```
Comment (dgarijo) [inline Assignment4/natalia figal- 22C051/copia_de_task07.py:51]: !! missing

**Review by dgarijo — CHANGES_REQUESTED** [review]
Half of the exersises are missing

**Comment by anaigmo** [comment]
Entregado y corregido, aunque falta algún ejercicio. No se admiten más PR.

**Pull request merged**